docs/reducing-contract-size/examples.md

```diff
@@ -0,0 +1,80 @@
+---
+sidebar_position: 1
+---
+
+# Advice & examples
+
+This page is made for developers familiar with lower-level concepts who wish to reduce their contract size significantly, perhaps at the expense of code readability.
+
+Some common scenarios where this approach may be helpful:
+
+- contracts intended to be tied to one's account management
+- contracts deployed using a factory
+- future advancements similar to the EVM on NEAR
+
+There have been a few items that may add unwanted bytes to a contract's size when compiled. Some of these may be more easily swapped for other approaches while others require more internal knowledge about system calls.
+
+## Small wins
+
+1. See the section in best practices on [creating smaller binaries](/best-practices#compile-smaller-binaries) using `opt-level` in the Cargo manifest.
+2. Ensure that your manifest doesn't contain `rlib` unless it needs to. Some NEAR examples have included this:
+
+  :::caution Possibly adds unnecessary bloat
+
+  ```toml
+  [lib]
+  crate-type = ["cdylib", "rlib"]
+  ```
+  :::
+
+  when it could be:
+
+  :::tip
+
+  ```toml
+  [lib]
+  crate-type = ["cdylib"]
+  ```
+  :::
+
+3. Avoid using the [`serde` crate](https://crates.io/crates/serde) and perhaps experiment with the [miniserde crate](https://crates.io/crates/miniserde) or the [rjson crate](https://crates.io/crates/rjson).
```

Comment:
Given that developers don't have an option to eject from this while using `near_bindgen` for JSON serialization, perhaps we should suggest using borsh serialization if not using cross contract calls? 

Also, I don't know if `rjson` or even `miniserde` are good suggestions even if they were supported as `rjson` will have some inconsistencies in certain cases and will be really hard for developers to match and a bad long term solution. In the case of `miniserde`, it might be alright if no customization is needed, but is extremely tedious when the default serialization derive is not sufficient, running into the same issues.

I think if we do suggest alternatives to use, we would need to mention it in the `no_std` section or indicate that this cannot be done with `near_bindgen` yet (or possibly ever since it is a very breaking change at this point)

Reply:
Commit here:
https://github.com/near/sdk-docs/pull/37/commits/9b7f5f163d93c2df330b1a771259c5277c795209